fsutils/ipcfg/ipcfg.h

```diff
@@ -49,7 +50,8 @@
 
 struct ipcfg_header_s
 {
-  uint8_t next;         /* Offset to the next IP configuration record */
+  uint8_t     next;     /* Offset to the next IP configuration record */
+  uint8_t     version;  /* For binary compatibility */
   sa_family_t type;     /* Either AF_INET or AF_INET6 */
 };
```

Comment:
This is really insufficient.  If the version number changes, then the size will also change.  This won't work in most cases.

Reply:
Nevermind.  I see that you ignore the data if the version number is bad.  That is okay if you don't try to read it.

Reply:
Yes. The ides is that returning to factory default are better than using corrupted data.